<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>434</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>                               Metadata Remover</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>331</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="selectFileButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectFolderButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>111</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Folder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cleanButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 255);</string>
       </property>
       <property name="text">
        <string>CLEAN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>70</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>451</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>29</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 255);
border: 2px solid black;</string>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">
border: 2px solid black;</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionreportissues"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <action name="actionreportissues">
   <property name="text">
    <string>Report Issues</string>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight Condensed</family>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
